redisjson/README.md

```diff
@@ -0,0 +1,70 @@
+<!--
+Copyright (c) 2014 - 2020 YCSB contributors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you
+may not use this file except in compliance with the License. You
+may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+implied. See the License for the specific language governing
+permissions and limitations under the License. See accompanying
+LICENSE file.
+-->
+
+## Quick Start
+
+This section describes how to run YCSB on RedisJson. 
+
+### 1. Start Redis with the RedisJSON module loaded (cf. https://oss.redislabs.com/redisjson/#building-and-loading-the-module)
+
+### 2. Install Java and Maven
+
+### 3. Set Up YCSB
```

Comment:
Please don't instruct folks to build/run the master branch. try something like this:

```

### 2. Set Up YCSB

Download the [latest YCSB](https://github.com/brianfrankcooper/YCSB/releases/latest) file. Follow the instructions.

```

Reply:
This is literally from the Redis README.md as part of the official YCSB.

> the client also needs to get included in `bin/bindings.properties` and `bin/ycsb` so that the run/load commands know what `redisjson` means.

Done. Do you need a new PR?

Reply:
> > the client also needs to get included in bin/bindings.properties and bin/ycsb so that the run/load commands know what redisjson means.
> Done. Do you need a new PR?

just push commits to the branch for this PR, please.

Reply:
> This is literally from the Redis README.md as part of the official YCSB.

yes, lots of old contributions had bad instructions in their README. We're trying to incrementally improve.